packages/std/src/timestamp.rs

```diff
@@ -37,62 +37,105 @@ impl Timestamp {
         Timestamp(Uint64::new(seconds_since_epoch * 1_000_000_000))
     }
 
+    /// Adds the given amount of days to the timestamp and
+    /// returns the result. The original value remains unchanged.
+    ///
+    /// Panicks if the result is exceeds the value range of [`Timestamp`].
     #[must_use = "this returns the result of the operation, without modifying the original"]
     #[inline]
     pub const fn plus_days(&self, addition: u64) -> Timestamp {
         self.plus_hours(addition * 24)
     }
 
+    /// Adds the given amount of hours to the timestamp and
+    /// returns the result. The original value remains unchanged.
+    ///
+    /// Panicks if the result is exceeds the value range of [`Timestamp`].
```

Comment:
```suggestion
    /// Panics if the result exceeds the value range of [`Timestamp`].
```
Same for the other ones. But nice, that these get some documentation now 👍